webui/react/src/components/kit/Select.tsx

```diff
@@ -0,0 +1,168 @@
+import { Select as AntdSelect } from 'antd';
+import type { DefaultOptionType, LabeledValue, RefSelectProps, SelectValue } from 'antd/es/select';
+import React, { forwardRef, useCallback, useMemo, useState } from 'react';
+
+import Label, { LabelTypes } from 'components/Label';
+import Icon from 'shared/components/Icon/Icon';
+
+import css from './Select.module.scss';
+
+const { OptGroup, Option } = AntdSelect;
+
+export { Option, SelectValue };
+
+type Options = DefaultOptionType | DefaultOptionType[];
+export interface Props<T = SelectValue> {
+  allowClear?: boolean;
+  defaultValue?: T;
+  disableTags?: boolean;
+  disabled?: boolean;
+  enableSearchFilter?: boolean;
+  filterOption?: boolean | ((inputValue: string, option: LabeledValue | undefined) => boolean);
+  filterSort?: (a: LabeledValue, b: LabeledValue) => 1 | -1;
+  id?: string;
+  label?: string;
+  maxTagCount?: 0 | 'responsive';
+  maxTagPlaceholder?: string;
+  mode?: 'multiple' | 'tags';
+  onBlur?: () => void;
+  onChange?: (value: T, option: Options) => void;
+  onDeselect?: (selected: SelectValue, option: Options) => void;
+  onSearch?: (searchInput: string) => void;
+  onSelect?: (selected: SelectValue, option: Options) => void;
+  options?: LabeledValue[];
+  placeholder?: string;
+  placement?: 'bottomLeft' | 'bottomRight' | 'topLeft' | 'topRight';
+  ref?: React.Ref<RefSelectProps>;
+  showSearch?: boolean;
```

Comment:
This prop can be removed because of `showSearch={!!onSearch || !!filterOption || showSearch}`. i think we need `onSearch` or `filterOption` anyways